contrib/systemd/btrbk.timer.in

```diff
@@ -1,9 +1,10 @@
 [Unit]
 Description=btrbk daily backup
+ConditionPathExists=|@CONFDIR@/btrbk/btrbk.conf
+ConditionPathExists=|@CONFDIR@/btrbk.conf
 
 [Timer]
-OnCalendar=daily
-AccuracySec=10min
```

Comment:
Not sure myself why I set this, must have read something when initially playing around with btrbk and systemd. Guess the idea was to tell the timer it's ok to be delayed a bit, as there might be other concurrent btrbk timers running (e.g. with different config and execution times)

Reply:
Well then it depends:<br>

Was your idea to spread the time when these fire off (to not cause all load at the same time)? Then `RandomizedDelaySec=` would be the appropriate setting, AFAIU.

Otherwise, I, personally would say, `1min` (default) would be enough.
Actually if you give system "so much time" just to fold multiple runs... it would rather cause more load (because more btrbks may run at the same time).

I could update the commit with whatever you like in the end. (The `OnCaleder` thing goes away of course)